### PR TITLE
Update docs for upcoming h5ad transition

### LINKED ISSUE #288: Update file list images for h5ad
In coordination with #287, we will need to update the following files in the `images` directory to change all mentions of `.hdf5` to `.h5ad`

- `anndata-project-download-folder.png`
- `anndata-sample-citeseq-download-folder.png`
- `anndata-sample-download-folder.png`
- `merged-anndata-project-citeseq-download-folder.png`
- `merged-anndata-project-download-folder.png`

This will finish docs updates related to https://github.com/AlexsLemonade/scpca-nf/issues/736

### PR DESCRIPTION
**Pull request by jashapiro**
I think I got all the places in the docs that we talked about hdf5 files.

We will still need to update the following files in the `images` directory
- `anndata-project-download-folder.png`
- `anndata-sample-citeseq-download-folder.png`
- `anndata-sample-download-folder.png`
- `merged-anndata-project-citeseq-download-folder.png`
- `merged-anndata-project-download-folder.png`

closes #288

### CODE REVIEW

**Comment by jashapiro** [comment]
Added the images from https://github.com/AlexsLemonade/scpca-docs/issues/288#issuecomment-2050381631

**Review by allyhawkins — APPROVED** [review]
LGTM

File: docs/download_files.md [hunk 1]
```diff
@@ -147,15 +147,15 @@ The [metadata file](#metadata) for multiplexed libraries (`single_cell_metadata.
...
+The object file, `SCPCP000000_merged.rds` or `SCPCP000000_merged_rna.h5ad`, contains both a raw and normalized counts matrix, each with combined counts for all samples in an ScPCA project.
 In addition to the counts matrices, the `SingleCellExperiment` or `AnnData` object stored in the file includes the results of library-weighted dimensionality reduction using both principal component analysis (PCA) and UMAP.
 See the {ref}`section on merged object processing<processing_information:merged objects>` for more information about how merged objects were created.
 
 If downloading a project that contains at least one CITE-seq library, the quantified CITE-seq expression data will also be merged.
 In `SingleCellExperiment` objects (`rds` files), the CITE-seq expression data is provided as an alternative experiment in the same object as the gene expression data.
-However, for `AnnData` objects, (`hdf5` files), the quantified CITE-seq expression is instead provided as a separate file called `SCPCP000000_merged_adt.hdf5`.
+However, for `AnnData` objects, (`hdf5` files), the quantified CITE-seq expression is instead provided as a separate file called `SCPCP000000_merged_adt.h5ad`.
```
Comment (jashapiro) [inline docs/download_files.md:158]:
```suggestion
However, for `AnnData` objects, (`h5ad` files), the quantified CITE-seq expression is instead provided as a separate file called `SCPCP000000_merged_adt.h5ad`.
```

File: docs/download_files.md [hunk 2]
```diff
@@ -52,22 +52,22 @@ See the [description of the Spatial transcriptomics output section below](#spati
 ### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
 ![sample download folder](images/anndata-sample-citeseq-download-folder.png)
 
-If downloading a sample that contains a CITE-seq library as an `AnnData` object (`hdf5` file), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.hdf5`.
+If downloading a sample that contains a CITE-seq library as an `AnnData` object (`hdf5` file), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
```
Comment (jashapiro) [inline docs/download_files.md:55]:
```suggestion
If downloading a sample that contains a CITE-seq library as an `AnnData` object (`.h5ad` file), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
```

**Comment by jashapiro** [comment]
@allyhawkins @dvenprasad I left references to "HDF5 files" in the text, but since the extension is now `.h5ad` everywhere, I worry that might be confusing. Should I change those to "H5AD files"?

**Comment by allyhawkins** [comment]
> @allyhawkins @dvenprasad I left references to "HDF5 files" in the text, but since the extension is now `.h5ad` everywhere, I worry that might be confusing. Should I change those to "H5AD files"?

I had seen this and was wondering the same thing, but assumed you had made the choice to keep them as HDF5. I think using H5AD everywhere would be less confusing though.

**Comment by allyhawkins** [comment]
We should wait to merge this until we have released the docs for merged objects. We will incorporate the new extension after merged objects are up.

**Comment by jashapiro** [comment]
@allyhawkins I requested rereview mostly just to confirm that we are ready to merge this.

**Review by allyhawkins — APPROVED** [review]
Yes we are good to go 👍

**Pull request merged**